### ISSUE TITLE
[UI] Cluster Templates - Hide "cattle-global-data:" in CT dropdown

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#21891]
Version: master-head (7/31/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Create one or more cluster templates
- Create a cluster, choose to do so via cluster templates, look at the drop-down

**Result:**
The drop down looks not very nice because of the mention of "cattle-global-data:" we should ideally just see the plain name of the revisions supplied for each cluster template. This will make it look better.

![image](https://user-images.githubusercontent.com/45179589/62251861-f43b1800-b3a5-11e9-8d3f-29189b8547bd.png)

**Comment by westlywright** [comment]
@davidnuzik This piggy-backs on a request vince had to break these out into two separate selects.  One select for cluster template and another for revision that is disabled until template is selected. I'll clean up the display as requested as well.

**Comment by davidnuzik** [comment]
Version: master-head (8/2/19) `bec76ef3`

This changes the design of how the user must pick their CT/CTR in the UI. The cattle-global-data is now also not shown. This is much cleaner:

![image](https://user-images.githubusercontent.com/45179589/62402938-35b2fb00-b53f-11e9-95e9-a4e7ca7c27db.png)

I tested and ensured I can create Clusters with the tempaltes without any issues. I created a CT with several revisions and ensured that the UI updates accordingly when change revisions and CT's. The HTTP/UI/API all look good.
